### PR TITLE
Add support for Redmi 6 Pro (xiaomi-sakura)

### PR DESCRIPTION
**Pull request by barni2000**
It works fine for me I can go in the bootloader menu and i can use MI A2 Lite variant of PMOS also.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ and then loaded by lk2nd.
 - Xiaomi Redmi 4 Prime - markw
 - Xiaomi Redmi Note 5 / 5 Plus Snapdragon - vince
 - Xiaomi Redmi S2 / Y2 - ysl
+- Xiaomi Redmi 6 Pro - sakura
 - Meizu M6 Note - m1721
 - Motorola Moto G7 Power - ocean
 - Xiaomi Mi A2 Lite - daisy
 - Xiaomi Mi A1 - tissot
-- XIaomi Mi A2 Lite - daisy
```
Comment (z3ntu) [inline README.md:38]: Why are you removing `daisy`?

File: dts/msm8953-xiaomi-common.dts [hunk 2]
```diff
@@ -29,7 +29,27 @@
 			};
 		};
 	};
-	
+
+	sakura {
+		model = "Xiaomi Redmi 6 Pro";
+		compatible = "xiaomi,sakura", "qcom,msm8953", "lk2nd,device";
+		lk2nd,match-bootloader = "*DAISY*";
```
[the remark this answers is not in the document]
Comment (alikates) [reply dts/msm8953-xiaomi-common.dts:36]: Its the same, just different stock OS

File: README.md (hunk 1 above)
Comment (alikates) [reply README.md:38]: It was duplicated

File: dts/msm8953-xiaomi-common.dts (hunk 2 above)
[the remark this answers is not in the document]
Comment (barni2000) [reply dts/msm8953-xiaomi-common.dts:36]: https://www.gsmarena.com/xiaomi_mi_a2_lite_(redmi_6_pro)-9247.php

[the remark this answers is not in the document]
Comment (z3ntu) [reply dts/msm8953-xiaomi-common.dts:36]: Once you add a comment that explains this, e.g. `/* Xiaomi Redmi 6 Pro (sakura) and Xiaomi Mi A2 Lite (daisy) are software compatible */` then this pull request is okay from my side.

**Review by z3ntu — APPROVED** [review]
LGTM

**Comment by alikates** [comment]
Wont't it match first with daisy and not use this device entry? Both models share the same board id and same bootloader string.
Looks like we cannot differenciate between daisy and sakura so isn't this redundant with daisys entry?

**Comment by barni2000** [comment]
> Wont't it match first with daisy and not use this device entry? Both models share the same board id and same bootloader string. Looks like we cannot differenciate between daisy and sakura so isn't this redundant with daisys entry?

daisy has different board id qcom,board-id= <0x1000b 0x09>, sakura's board id same with mido and tissot qcom,board-id = <0x1000b 0> that cause this issue https://github.com/msm8953-mainline/linux/issues/82. 
It always match for mido first because mido hasn't got lk2nd,match-bootloader entry.

**Comment by alikates** [comment]
Okay, LGTM

**Pull request merged**